cetmix_tower_ovh/__manifest__.py

```
@@ -0,0 +1,24 @@
+    "website": "https://tower.cetmix.com",
+    "images": ["static/description/banner.png"],
+    "application": False,
+    "installable": True,
+    "demo": [
+        "demo/demo_data.xml",
+    ],
+    "external_dependencies": {"python": ["ovh"]},
+    "depends": [
+        "cetmix_tower_server",
+    ],
+}
```

Pylint (3.3.7)

[warning] Statement seems to have no effect

(W0104)